server/repository/src/migrations/mod.rs

```diff
@@ -22,7 +23,9 @@ use thiserror::Error;
 
 pub(crate) trait Migration {
     fn version(&self) -> Version;
-    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()>;
+    fn migrate(&self, _: &StorageConnection) -> anyhow::Result<()> {
```

Comment:
Added default so that when postgres specific migrations are done we don't have to create empty migrate method for sqlite